scalafix/rules/src/main/scala/rsc/rules/semantics/Scopes.scala

```diff
@@ -12,6 +12,8 @@ import scala.meta.internal.semanticdb.Scala.{Names => n}
 sealed trait Scope {
   def lookup(name: n.Name): String
 
+  def lookupRename(name: n.Name): String = ""
```

Comment:
I think we want to not use `lookup` and add a comment. This won't be returning a symbol, but just an unqualified name.

Reply:
I like your getRename suggestion and will push with this in a few minutes.